packages/stark-ui/src/util/form/form.util.ts

```diff
@@ -62,9 +62,9 @@ export class StarkFormUtil {
 	 * @param formGroup - Angular form group object
 	 * @param statesToBeSet - States to be set to the form
 	 */
-	public static setFormGroupState(formGroup: FormGroup, statesToBeSet: StarkFormState[]): void {
+	public static setUntypedFormGroupState(formGroup: UntypedFormGroup, statesToBeSet: StarkFormState[]): void {
 		// Verifying it is indeed an Angular FormGroup
-		if (StarkFormUtil.isFormGroup(formGroup)) {
+		if (StarkFormUtil.isUntypedFormGroup(formGroup)) {
```

Comment:
Could you rename this back to `setFormGroupState` ?

Reply:
done